include/flatbuffers/idl.h

```diff
@@ -59,6 +59,21 @@ namespace flatbuffers {
   TD(STRUCT, "",       Offset<void>, int, int, int, int) \
   TD(UNION,  "",       Offset<void>, int, int, int, int)
 
+#define FLATBUFFERS_GEN_TYPES_SCALAR_ALIAS(TD) \
```

Comment:
Why not stick this information in FLATBUFFERS_GEN_TYPES_SCALAR ? less duplication that way, and probably simpler code.